### DISCUSSION TITLE
YaoAPI incompatibale

### DISCUSSION THREAD

**Discussion by Saxxxon** [QuantumBFS/Yao.jl#344] [Q&A]
Yao 0.7.0 requires YaoAPI 0.3 but the latest version on gihub is only 0.1.0

**Comment by GiggleLiu** [comment]
The latest version of YaoAPI is not 0.1, check the one in Yao.jl, folder lib/YaoAPI. Maybe you are refereing to repo YaoAPI.jl, this one has been archived.

**Comment by Saxxxon** [comment]
![image](https://user-images.githubusercontent.com/86470076/157768025-da6e5cb8-a79b-445b-a7a3-54e0d8acef79.png)
I wonder how should I fix this, thank you!

**Comment by Roger-luo** [comment]
Have you updated your registry? You can update the registry via Pkg.update()

**Comment by Saxxxon** [comment]
![image](https://user-images.githubusercontent.com/86470076/157774408-5207c46a-05c2-42d7-9c8c-c308b7746861.png)
I don't think it is working.

**Comment by GiggleLiu** [comment]
https://github.com/JuliaRegistries/General/blob/master/Y/YaoAPI/Versions.toml

There should be something wrong with your registry. Maybe try removing YaoAPI and related packages first.

**Comment by Roger-luo** [comment]
I'll convert this to a discussion since I don't think this is related to Yao itself. But please feel free to provide more information to let us help you if you it still not working
